### DISCUSSION TITLE
New Insights Hub Root CA certificate

### DISCUSSION THREAD

**Discussion by MOCEDRIC** [mindsphere/node-red-contrib-mindconnect#183] [General]
The current Insights Hub Root CA certificate will be replace with a DigiCert Root CA certificate on October 15, 2024, at 04:00 UTC. 
This change will only impact those utilizing MindConnect API-based agents.
Is the [node-red-contrib-mindconnect] impacted?
Thanks in advanced

**Comment by sn0wcat** [comment]
@MOCEDRIC  theoretically the agents should be able to deal with it, however the rotation of certificates has only be tested once in history of the insights hub, so I can't say how it will go this time. Maybe the agents will need to be onboarded again. Changing root certificate sounds to me like asking for trouble to be honest ;)

**Comment by siemens-developer** [comment]
@MOCEDRIC The certificate update is for change of CA from QuoVadis to DigiCert. As both are standard and very common certifying authorities, most of the systems have both certificates in the trusted certificate store, so there should be minimal impact for clients who use the standard Operating Systems and high-level programming languages. 
We have verified node-red client on our test environment where there was no impact for the agent as the certificate is not hard coded in source code.  ref link: https://nodejs.org/api/tls.html#tlscreatesecurecontextoptions
